src/mult.jl

```diff
@@ -60,7 +73,7 @@ end
 function _term_poly_mult(t::Term{C, S}, p::Polynomial{C, T}, op::Function) where {C, S, T}
     U = MA.promote_operation(op, S, T)
     if iszero(t)
-        zero(Polynomial{C, U})
+        zero( Polynomial{C,U} )
```

Comment:
Remove spaces before and after parenthesis to stay consistent with the rest of the code base